promscale/tobs/tobs.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Install the observability stack (tobs) for Kubernetes
+excerpt: Install tobs
+product: promscale
+keywords: [analytics, Kubernetes]
+tags: [install, k8s]
+related_pages:
+  - /promscale/:currentVersion:/recommendations/resource-recomm
+  - /promscale/:currentVersion:/send-data/
+---
+
+import PromscaleConfigurationRecommendations from 'versionContent/_partials/_promscale-configuration-recommendations.mdx';
+
+# Install the observability stack (tobs) for Kubernetes
+The observability stack (tobs), is a pre-packaged distribution of observability
+tools and dashboard interfaces which can be installed on any existing Kubernetes
+cluster. It includes many of the most popular open-source observability tools
+including Prometheus, Grafana, Jaeager, TimescaleDB, and [other components][stack-components].
+Together, these provide a maintainable solution to analyze the traffic on the
+server and identify any potential problems with a deployment. You can use Helm
+charts to configure and update [tobs][tobs] deployments. 
+
+ 
+## Before you begin
+*  Understand the [resource requirements][resource-requirements] for your cluster.
+*  [Deploy a Kubernetes Cluster][kubernetes-cluster]. 
+*  Install [kubectl][kubectl] in your local environment and
+   [connect to your cluster][connect-to-cluster].
+*  Install [Helm 3][helm] in your local machine. 
+
+## Installing Promscale with tobs 
+The observability stack (tobs), eliminates the need to maintain configuration
+details for each of the applications, while providing observability for the
+applications running on your cluster. You can deploy tobs on your cluster and
+access it locally with `kubectl` [Port-Forward][port-forward].
+
+<procedure>
+
+1.  Create the `observability` namespace on your cluster:
+    ```bash
+    kubectl create namespace observability
+    ```   
+1. Install a certificate manager for your cluster:
+   ```bash
+   kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.8.0/cert-manager.yaml
```

Comment:
This is just one command step to install cert-manager to avoid the churn the command was included here. However, I will a sentence about for updated instructions, see"xyz". A note about open telemetry will also be added.